spec/models/group_spec.rb

```diff
@@ -94,7 +94,36 @@
           @discussion.destroy
         }.to change { @group.reload.motions_count }.by(-1)
       end
+    end
+
+    describe "#closed_motions_count" do
+      before do
+        @user = create(:user)
```

Comment:
I used this syntax because it's consistent with the other counter cache specs. Should I change it?